src/Command/CommandBase.php

```diff
@@ -1571,6 +1571,14 @@ protected function createMySqlDumpOnLocal(string $dbHost, string $dbUser, string
     return $localFilepath;
   }
 
+  protected function invokeCurlCommand(string $command, Closure $outputCallback = NULL): void {
+    $this->localMachineHelper->checkRequiredBinariesExist(['curl']);
+    $process = $this->localMachineHelper->executeFromCmd($command, $outputCallback, NULL, ($this->output->getVerbosity() > OutputInterface::VERBOSITY_NORMAL));
```

Comment:
Can you use localMachineHelper->execute() instead of localMachineHelper->executeFromCmd()?

As described in the method comments, executeFromCmd should only be used as a last resort.

Reply:
I tried using execute only initially but faced issue (https://acquia.atlassian.net/browse/GL-2368?focusedCommentId=8264421)for same input it did worked with executeFromCmd.